docs/src/components/ReanimatedAnimations/index.tsx

```diff
@@ -0,0 +1,170 @@
+    body: 'Connect your animations to a gyroscope or accelerometer with just one hook. It’s that easy.',
+    code: `const gyroscope = useAnimatedSensor(SensorType.GYROSCOPE);
+
+    useDerivedValue(() => {
+      const { x, y, z } = gyroscope.sensor.value;
+    });`,
+    component: <GyroscopeExample />,
+    docsLink:
+      'https://docs.swmansion.com/react-native-reanimated/docs/device/useAnimatedSensor',
+  },
+  {
+    title: 'Keyboard-based animations',
+    body: 'Create animations based on the device keyboard state and position.',
+    code: `function App() {
+      const keyboard = useAnimatedKeyboard();
+      const animatedStyle = useAnimatedStyle(() => ({
+        transform: [{ translateY: -keyboard.height.value }],
+      });
+      //...
+     }`,
+    component: <KeyboardExample />,
+    docsLink:
+      'https://docs.swmansion.com/react-native-reanimated/docs/device/useAnimatedKeyboard',
+  },
+  {
+    title: 'Shared Element Transitions',
+    body: 'Seamlessly animate elements between navigation screens with a single line of code.',
+    code: `function App() {
+      <Animated.View
+       sharedTransitionTag="hero-element"
+      />
+    }`,
+    component: <SharedElementExample />,
+    docsLink:
+      'https://docs.swmansion.com/react-native-reanimated/docs/shared-element-transitions/overview',
+  },
+];
+
+const ReanimatedAnimations = () => {
+  const [activeSection, setActiveSection] = useState(-1);
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (!containerRef.current) return;
+
+      const containerTop = containerRef.current.getBoundingClientRect().top;
+
+      const sectionHeight = window.innerHeight;
+
+      const currentSection =
+        containerTop < 100 &&
+        Math.abs(containerTop) < (sections.length - 0.75) * sectionHeight
+          ? Math.floor(Math.abs(containerTop) / (sectionHeight - 75))
+          : -1;
+
+      setActiveSection(currentSection);
+    };
+
+    window.addEventListener('scroll', handleScroll);
+    handleScroll();
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, []);
+  return (
+    <ReanimatedAnimationsBackground>
+      <div className={styles.animationsContainer}>
+        <div>
+          <h3>Play with the animations!</h3>
+          <h5>
+            Check out the animations and check how you can implement it to your
+            product.
+          </h5>
+        </div>
+        <div ref={containerRef}>
+          {sections.map((section, idx) => (
+            <Fragment key={idx}>
+              <ReanimatedAnimationsSection
+                title={section.title}
+                body={section.body}
+                code={section.code}
+                component={section.component}
+                docsLink={section.docsLink}
+                mobileComponent={section.mobileComponent ?? null}
+                tabletComponent={section.tabletComponent ?? null}
```

Comment:
I don't get this. If its nullish then use `null`? Is the `something ?? null` necessary?